setup.py

```diff
@@ -19,7 +19,10 @@
     "scipy>=1.8",
     "qiskit>=0.36.1",
     "pyquil>=3.1.0",
-    "docplex>=2.23.1"
+    "docplex>=2.23.1",
+    "autograd~=1.4",
```

Comment:
Are these requirements from pennylane?

If possible, I'd turn everything into `>=` so that we are minimising the chances of being locked in the dependencies madness

Reply:
Yes, this are requirements for the pennylane files.

Changed to `>=`